src/App.tsx

```diff
@@ -28,15 +38,84 @@ const MapListScreen = () => {
   return <MapList></MapList>;
 };
 
+//Created dummy About Us screen to test all three tabs
+const AboutUsScreen = () => {
+  return (
+    <View
+      style={{
+        flex: 1,
+        alignItems: "center",
+        justifyContent: "center",
+        backgroundColor: "#ffffff",
+      }}
+    >
+      <Text style={{ fontSize: 30 }}>About Us</Text>
+    </View>
+  );
+};
+
+// If the focused route is not found, we need to assume it's the initial screen
+// This can happen during if there hasn't been any navigation inside the screen
+// In our case, it's "Home" as that's the first screen inside the navigator
+const getHeaderTitle = (route) => {
+  const routeName = getFocusedRouteNameFromRoute(route) ?? "Home";
+
+  switch (routeName) {
+    case "Home":
+      return "Home";
+    case "Map":
+      return "MapList";
+    case "About Us":
+      return "About Us";
+  }
+};
+
+const Tabs = ({ navigation, route }) => {
```

Comment:
These type errors are still present, PR should not be merged until they are corrected.

Reply:
@rweitzman I have a generic fix for the type errors and consolidating the types into a separate file; maybe we can merge it as a separate PR.